checkmk_extensions/agent/robotmk_collector.ps1

```diff
@@ -3,9 +3,26 @@ $ErrorActionPreference = "Stop"
 
 function Main {
     Param (
-        [parameter(Mandatory=$true, Position=0)]
+        [parameter(Mandatory=$false, Position=0)]
```

Comment:
Since we are never calling `Main` with a parameter, we can remove it alltogether